packages/fxa-auth-server/scripts/apple-transfer-users.js

```diff
@@ -35,10 +36,16 @@ if (!program.input) {
 }
 
 async function main() {
-  const migration = new ApplePocketFxAMigration(program.input, config, AuthDB, program.output, program.delimiter);
+  const migration = new ApplePocketFxAMigration(program.input, config, AuthDB, program.output, program.delimiter, program.mock);
   await migration.load();
   await migration.transferUsers();
   await migration.close();
+
+  // For very large lists, we need to comment this out
```

Comment:
Does this mean some async process hasn't finished by the time process.exit is called?